docs/cloudsync_mqtt.md

```diff
@@ -0,0 +1,272 @@
+# Cloud Synchronization
+## Contents
+1. [Introduction](#1-introduction)
+2. [Launching and Configuring EC2 Instance](#2-EC2-Instance-Setup-Description)
+3. [Installing MQTT Broker on AWS](#3-MQTT-Broker-Installation-on-AWS)
+4. [Certificate Generation for MQTT](#4-Certificates-Generation-for-MQTT)
+5. [Configurations for MQTT broker](#5-MQTT-Configurations)
+6. [Run CloudSync](#6-Execute-Cloudsync)
+
+# 1. Introduction
+This module would be responsible for sending the data (can be sensor data or any reading or image/video data, etc.) collected from different devices in the home environment to a cloud Endpoint (AWS). This Sync to the cloud will be using a API Call by the service application. The broker used is mosquitto which is setup at AWS endpoint. The Service APplication will make a POST API call which will then check first for the connection to the broker and then establish a connection if not present to further publish the data to the cloud. After successful publish or in case of failure corresponding response is sent back to the service application<br><br>
```

Comment:
```suggestion
This module would be responsible for sending the data (can be sensor data or any reading or image/video data, etc.) collected from different devices in the home environment to a cloud Endpoint (AWS). This Sync to the cloud will be using a API Call by the service application. The broker used is mosquitto which is setup at AWS endpoint. The Service Application will make a POST API call which will then check first for the connection to the broker and then establish a connection if not present to further publish the data to the cloud. After successful publish or in case of failure corresponding response is sent back to the service application<br><br>
```

Reply:
Done!!!